css_tasks/task9/scss/_jumbotron.scss

```diff
@@ -0,0 +1,206 @@
+@import "variables";
+@import "mixins";
+
+.jumbotron {
+  position: relative;
+  &__image {
+    width: 100%;
+    max-height: 40rem;
+    opacity: 0.82;
+    background: #000;
+  }
+  &__content {
+    position: absolute;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-column-gap: 2rem;
+    top: 13.16%;
+    .jumbotron-content__left-column {
+      @include flexColumn;
+      .jumbotron-header__title {
+        @include jumbotronTitle;
+        padding-bottom: 0.75rem;
+        font-size: $jumbotronTitleFontSize;
+        span:first-child {
+          @include logoFirstSpan;
+        }
+        span:last-child {
+          @include logoSecondSpan;
+        }
+      }
+      .jumbotron-header__subtitle {
+        @include sectionHeader;
+        color: $sectionColor;
+      }
+      .jumbotron-left-column__separator {
+        width: 14%;
+        height: 2px;
+        margin-top: 1.5rem;
+        background: $sectionColor;
+      }
+      .jumbotron-left-column__text {
+        margin-top: 1.5rem;
+        font-family: $secondaryFont;
+        font-size: $jumbotronTextFontSize;
+        color: $navbarColor;
+        line-height: 1.44;
+      }
+      .jumbotron-left-column__button {
```

Comment:
compiled selector for this button looks like `.jumbotron__content .jumbotron-content__left-column .jumbotron-left-column__button` without no reason here is created so big specificity

Reply:
👍